app/src/main/java/com/kickstarter/viewmodels/ProjectViewModel.kt

```diff
@@ -247,6 +247,9 @@ interface ProjectViewModel {
 
         /** Emits when we should update the [com.kickstarter.ui.fragments.BackingFragment] and [com.kickstarter.ui.fragments.RewardsFragment].  */
         fun updateFragments(): Observable<ProjectData>
+
```

Comment:
No new logic should be added on ProjectViewModel just for triggering the new event on Manage pledge screen

Reply:
done